app/assets/stylesheets/_footer.scss

```diff
@@ -1,27 +1,25 @@
-$footer-height: 30px;
+$footer-height: 40px;
 
-html, body {
-  /* The html and body elements cannot have any padding or margin. */
-  height: 100%;
+html {
+  position: relative;
+  min-height: 100%;
 }
 
-.wrapper {
-  min-height: 100%;
-  height: auto;
-  //margin: 0 auto -($footer-height + 1px) 0; // https://github.com/nex3/sass/issues/1023
-  margin: 0 auto 0;
-  margin-bottom: -($footer-height + 1);     // +1 to account for the 1px border in the footer.
-  padding-bottom: $footer-height + 10px;    // The extra prevents content from bumping right into our footer.
+body {
+  /* Margin bottom by footer height */
+  margin-bottom: $footer-height;
 }
 
-footer {
+.footer {
+  position: absolute;
+  bottom: 0;
+  width: 100%;
   height: $footer-height;
```

Comment:
Bootstrap applies `box-sizing: border-box` to all elements, so that means the border is included in the height of the element. Since we are specifying a height of 40px and a border of 1px, that means the contents of the element are 39px. But the line-height is specified as 40px, which is bigger than the content area. This causes a layout problem which results in a vertical scrollbar appearing on the page.

To fix this layout problem I would change the height:

```
height: $footer-height + 1px;
```

or change the line-height: 

```
line-height: $footer-height - 1px;
```